README.md

```diff
@@ -37,7 +37,9 @@ The fields in the table below can be used in these parts of STAC documents:
 
 | Field Name       | Type                     | Description |
 | ---------------- | ------------------------ | ----------- |
-| proj:epsg        | integer\|null   | [EPSG code](http://www.epsg-registry.org/) of the datasource |
+| proj:epsg        | integer\|null   | [EPSG code](http://www.epsg-registry.org/) of the datasource; Maintained for backwards compatability and will be deprecated in V2.0.0. Please use `proj:authority` and `proj:code`. |
+| proj:code        | string\|null   | Authority specific code of the data source (e.g., [EPSG](http://www.epsg-registry.org/), [IAU](http://voparis-vespa-crs.obspm.fr:8080/web/2015.html)) |
```

Comment:
We link the OGC url instead of the IAU URL.

https://github.com/opengeospatial/NamingAuthority/issues/212

Reply:
Updated to link to the OGC site. I also did not add a table for an enum as I am not sure what the authority would map to. Instead, I incldued a link to the OGC where they identify 4 different authorities (OGC, EPSG, COSMO, and IAU). If I needed to create a table, I would likely link IAU to http://www.opengis.net/def/crs/IAU and EPSG to http://www.opengis.net/def/crs/EPSG, so I am not sure that a table is a value add here.